src/test/ui/issues/issue-13497-2.stderr

```diff
@@ -1,10 +1,7 @@
 error[E0515]: cannot return value referencing local variable `rawLines`
   --> $DIR/issue-13497-2.rs:3:5
    |
-LL |        rawLines
-   |  ______^
-   | | _____|
-   | ||
+LL | //     rawLines
 LL | ||         .iter().map(|l| l.trim()).collect()
    | ||_______________-___________________________^ returns a value referencing data owned by the current function
    |  |_______________|
```

Comment:
It'd be nice to tweak the layout algorithm here to avoid the first unnecessary overlap, but it might be tricky to get right.

<img width="821" alt="Screen Shot 2022-12-27 at 11 06 45 AM" src="https://user-images.githubusercontent.com/1606434/209711815-602e8912-d963-4ccd-9caa-408c15cc7298.png">